tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/tabbed_paragraph_block.html

```diff
@@ -0,0 +1,76 @@
+{% load wagtailcore_tags %}
+
+<div class="tabbed-paragraph grid__tabbed-paragraph">
+
+    {% include "patterns/atoms/motif-heading/motif-heading.html" with heading_level=2 heading=value.title classes="motif-heading--two motif-heading--static tabbed-paragraph__heading" %}
+
+    <p class="tabbed-paragraph__intro">{{ value.intro }}</p>
+
+    {# desktop - tabs #}
```

Comment:
This may have been discussed in a meeting but wondering if there should be a no-js fallback for desktop?

Reply:
Oh you are right, I forgot to add that in. Will have a think about how to do this, but it might need to be a follow-up ticket now.

Reply:
So I just discovered https://developer.mozilla.org/en-US/docs/Web/CSS/@media/scripting for a quick and dirty fix. Doesn't look lovely but the key thing is that the content is all visible without js.

![Screenshot 2024-02-27 at 10 21 50](https://github.com/torchbox/torchbox.com/assets/771869/4d2c250d-0a0b-4fd9-a317-ddb18ed3ce9c)